File: FrameIncam.Domains/Models/Master/FreeLancer/MasterFreeLancerPackage.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace FrameIncam.Domains.Models.Master.FreeLancer
{
    [Table("master_freelancer_package")]
    public class MasterFreeLancerPackage : LogModel
    {
        [Column("freelancer_id")]
        public int FreeLancerId { get; set; }

        [Column("description")]
        public string Description { get; set; }
        [Column("price_per_day")]
        public decimal PricePerDay { get; set; }
    }
}

File: FrameIncam.Client/ng/domains/master/freelancer/masterFreeLancerPackage.js
(function ($, ng, debugMode, undefined) {
    "use strict";

    // --------------------------------------------------------------------------------
    // Dependencies
    // --------------------------------------------------------------------------------
    var module = ng.module("app");

    // --------------------------------------------------------------------------------
    // Model
    // --------------------------------------------------------------------------------
    module.run([
        "services.utils",
        "services.models",
        function (
            p_utils,
            p_models
        ) {
            // --------------------------------------------------------------------------------
            // Namespace
            // --------------------------------------------------------------------------------
            var fincam = p_models.namespace("fincam");
            var freelancer = p_models.namespace("fincam.master.freelancer");

            // --------------------------------------------------------------------------------
            // Model
            // --------------------------------------------------------------------------------
            p_models.class({
                namespace: freelancer,
                name: "masterFreeLancerPackage",
                base: {
                    namespace: fincam,
                    name: "logModel"
                },
                properties: {
                    id: "int",
                    freeLancerId: "int",
                    description: "string",
                    pricePerDay: "decimal"
                },
                prototype: new function () {
                    var self = this;
                    return self;
                }
            });
        }
    ]);

})(jQuery, angular, document.children[0].hasAttribute("debug"));